nixos/modules/services/x11/window-managers/xinitrc.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.xinitrc;
+in
+
+{
+  options = {
+    services.xserver.windowManager.xinitrc = {
+      enable = mkOption {
```

Comment:
Use `mkEnableOption` instead, this is just a short-cut over the boiler plate of `enable = mkOption { …`.